src/Framework/Framework/Controls/KnockoutHelper.cs

```diff
@@ -351,7 +343,7 @@ private static JsExpression TransformOptionValueToExpression(DotvvmBindableObjec
             }
         }
 
-        private const string RootValidationTargetExpression = "dotvvm.viewModelObservables['root']";
+        private const string RootValidationTargetExpression = "/";
```

Comment:
At this point, I think we can just inline this constant.

Reply:
Agree. Inlined.